app/models/feed.rb

```diff
@@ -16,6 +16,7 @@ class Feed < ApplicationRecord
   scope :expired, -> { where.not(expired_at: nil) }
   scope :stale, -> { where("updated_at < ?", 3.months.ago) }
   scope :throttled, -> { where.not(throttled_at: nil) }
+  scope :unthrottleable, -> { where("throttled_at < ?", 3.days.ago) }
```

Comment:
Maybe we should save some information about rejected emails so this can be 3 days _and_ `week_count >= week_limit`?

Reply:
you're right, we should only unthrottle if it's under the limit 👍🏻